mdoc/Test/en.expected-frameworks/FrameworksIndex/One.xml

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Framework Name="One">
+  <Type Name="MyFramework.MyNamespace.MyClass">
+    <Member Sig="public double OnlyInClassic { get; set; }" />
```

Comment:
I also half wonder if we should use IL for the signature, not C#, because C# can/will change as new features are added to the language.

For example, C# 7 tuples:

    public (int start, int end) GetExtents () {...}

There's two -- compatible! -- ways to render this for doc purposes:

    // mdoc doesn't know wtf tuples are yet...
    public ValueTuple<int, int> GetExtents();

    // mdoc *does* know wtf tuples are
    public (int start, int end) GetExtents();

So long as everything is updated with the same tooling at the same time, and is thus internally consistent, things will work -- updating the C# signature generator to properly support tuples won't "break" anything.

*As soon as* that's not the case -- e.g. documentation comes from multiple different sources and is "integrated" at runtime, which the MonoDoc GUI tries to do on macOS -- then you may encounter problems.

Offhand, I'm not sure if this "frameworks" support is something that could plausibly support "merge at runtime from various disparate sources" behavior.

If it *can*, that means we need a *stable*, *unchanging* signature language.

*C# is not that language*. IL *is*.